#request fields
int32 group_number
string ip_address
---
#response fields
int32 state
